SECURITY.md

```diff
@@ -0,0 +1,83 @@
+## Configuring Security
+
+
+The following variables configure container security using the socialLogin-1.0 feature.  
+
+### Configuration needed at image build time:
+
+ * The environment variable `sec_sso_providers` must be defined and contain a space delimited list of the identity providers to use. If more than one is specified, the user will be able to choose which one to authenticate with. Valid values are any of `oidc oauth facebook twitter github google linkedin`.  Specify `ARG sec_sso_providers="(your choice goes here)"` in your Dockerfile.
+
+ * Providers usually require the use of HTTPS.  Specify `ARG TLS=true` in your Dockerfile. 
+
+ * To automatically trust certificates from well known identity providers, specify  `ARG SEC_TLS_TRUSTDEFAULTCERTS=true` in your Dockerfile.
+
```

Comment:
should specify here that for these env vars to take effect the user needs to call `configure.sh`

Reply:
fixed